source/plugin/metrics.rst

```diff
@@ -0,0 +1,67 @@
+==================
+Metrics Collection
+==================
+
+.. javadoc-import::
+    org.spongepowered.api.Sponge
+    org.spongepowered.api.plugin.PluginContainer
+    org.spongepowered.api.util.metric.MetricsConfigManager
+
+Collection of metrics from servers that use your plugin can be an incredibly useful tool for knowing what direction
+to take your plugin in. However, the :doc:`/ore/guidelines` (under External Connections) state that plugins may
+collect metrics if informed consent has been granted. Such consent must be **opt-in**, metrics collection must be off
```

Comment:
I think there is an "only" missing, as in "... may *only* collect metrics if ..."